documentation/modules/configuring/con-maintenance-time-window-definition.adoc

```diff
@@ -3,23 +3,41 @@
 // assembly-maintenance-time-windows.adoc
 
 [id='con-maintenance-time-window-definition-{context}']
-= Maintenance time window definition
+= Scheduling maintenance time windows
 
-You configure maintenance time windows by entering an array of strings in the `Kafka.spec.maintenanceTimeWindows` property.
-Each string is a {CronExpression} interpreted as being in UTC (Coordinated Universal Time, which for practical purposes is the same as Greenwich Mean Time).
+[role="_abstract"]
+Schedule updates by the Cluster Operator to Kafka or ZooKeeper clusters for minimal impact on client applications.
 
-The following example configures a single maintenance time window that starts at midnight and ends at 01:59am (UTC), on Sundays, Mondays, Tuesdays, Wednesdays, and Thursdays:
+Updates are usually triggered by changes to the `Kafka` resource.
```

Comment:
Can we make it clear it clear that these changes are either done by the user or user's tooling?